torchaudio/functional/filtering.py

```diff
@@ -1,4 +1,5 @@
 import math
+import os
```

Comment:
I think this is not used, and since we landed #1214, this will cause style check to fail if landed on master. Can you remove and merge/rebase the latest master?